using_images/xpaas_images/fuse.adoc

```diff
@@ -65,23 +63,32 @@ The Maven Archetype catalog includes the following examples:
 
 Begin by selecting the archetype which matches the type of application you would like to create.
 
-==== Simple Workflow to Create an Application from Maven Archetype Catalog
+=== Create an Application from the Maven Archetype Catalog
 
-. Use the Maven archetype catalog to create a sample project with the required resources. In order to use archetypes it is important to configure the Maven repositories which holds the archetypes and artifacts you may need:
+You must configure the Maven repositories, which holds the archetypes and artifacts you may need, before creating a sample project:
 
 * JBoss Fuse repository: `https://repo.fusesource.com/nexus/content/groups/public/`
 * RedHat GA repository: `https://maven.repository.redhat.com/ga`
-+
-You can then use the Maven archetype catalog to create a sample project with the required resources. For example, the following command will create a sample project:
+
+[start=1]
+. Use the maven archetype catalog to create a sample project with the required resources. The command to create a sample project is:
```

Comment:
This is really a 1-step procedure, which we don't really do. I'd recommend that you lose the numbering entirely.